src/funds/funds_meta.py

```diff
@@ -0,0 +1,291 @@
+
+        Returns:
+            FundMeta: FundMeta entry
+        """
+        group = self.get_fund_group(fund_code=code)
+        fund_id = self.get_fund_id(fund_code=code)
+        values_basic = self.get_fund_values_basic_for_fund_id(fund_id=fund_id)
+        price_date = self.parse_csv_date(values_basic["price_date"])
+        a_share_value = self.parse_csv_float(values_basic["a_share_value"])
+        b_share_value = self.parse_csv_float(values_basic["b_share_value"])
+        _1d_change = self.parse_csv_float(values_basic["1d_change"])
+        _1m_change = self.parse_csv_float(values_basic["1m_change"])
+        _1y_change = self.parse_csv_float(values_basic["1y_change"])
+        _3y_change = self.parse_csv_float(values_basic["3y_change"])
+        _5y_change = self.parse_csv_float(values_basic["5y_change"])
+        _10y_change = self.parse_csv_float(values_basic["10y_change"])
+        _15y_change = self.parse_csv_float(values_basic["15y_change"])
+        _20y_change = self.parse_csv_float(values_basic["20y_change"])
+        profit_projection = self.parse_csv_float(values_basic["profit_projection"])
+        profit_projection_date = self.parse_csv_date(values_basic["profit_projection_date"])
+
+        return FundMeta(
+                        id=self.create_id(fund_code=code),
+                        fund_code=code,
+                        fund_id=fund_id,
+                        color=fund_json_entry["color"],
+                        kiid=fund_json_entry.get("kiid", None),
+                        long_name=fund_json_entry["longName"],
+                        name=fund_json_entry["name"],
+                        risk=fund_json_entry["risk"],
+                        short_name=fund_json_entry["shortName"],
+                        subs_name=fund_json_entry.get("subsName", None),
+                        group=self.group_map[group["group"]],
+                        price_date=price_date,
+                        a_share_value=a_share_value,
+                        b_share_value=b_share_value,
+                        _1d_change=_1d_change,
+                        _1m_change=_1m_change,
+                        _1y_change=_1y_change,
+                        _3y_change=_3y_change,
+                        _5y_change=_5y_change,
+                        _10y_change=_10y_change,
+                        _15y_change=_15y_change,
+                        _20y_change=_20y_change,
+                        profit_projection=profit_projection,
+                        profit_projection_date=profit_projection_date
+                      )
+
+    def parse_csv_date(self, str: str) -> Optional[date]:
+        """Parses date from CSV value
+
+        Args:
+            str (str): CSV value
+
+        Returns:
+            Optional[date]: date
+        """
+        if "-" == str:
+            return None
+
+        return datetime.strptime(str, "%d.%m.%Y").date()
+
+    def parse_csv_float(self, str: str) -> Optional[float]:
```

Comment:
here also parameter name should be different from a type. And in this case("str") we are talking of a builtin name too.